sharding/sharding.go

```diff
@@ -119,8 +119,11 @@ func (r *ShardingFerry) Run() {
 	cutoverStart := time.Now()
```

Comment:
How can we accurately report downtime now? This is going to include all retries which is not great

Reply:
Good catch. Would moving this line into the callback func below (after line 122) with a var declaration outside of the block suffice as it will reset with each `POST`?

Reply:
I'm not sure what end result we want to see on our dashboard, I defer to @milanatshopify

Reply:
We want to see the total time the shop was down during the shop move. That's probably a sum of individual ones, as that is relevant for the customers, though I can see some value in emitting multiple values.